frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -56,38 +56,33 @@ type SegmentInfo = {
 };
 
 const exportStatisticsToCSV = (
-  segmentInformation: Array<SegmentInfo> | string,
+  segmentInformation: Array<SegmentInfo> | null,
```

Comment:
Please change it like this:

```suggestion
  segmentInformation: Array<SegmentInfo>,
```

and then adapt the caller so that the function is not called in the null case.